src/test/java/com/griddynamics/examples/qa/student/practice/CalculatorTest.java

```diff
@@ -22,28 +22,31 @@
   @Test(dataProvider = "dataProvider", groups = "add")
   @Stories("Add test")
   public void addTest(int a, int b) {
-    System.out.println("Add test with values (" + a + ", " + b + ")");
-    Assert.assertEquals(a + b, Calculator.add(a, b), String.format(
-        "Incorrect value returned from add method with parameters ($d , $d)", a, b));
+    baseTest.logging(String.format(
```

Comment:
Would you mind to change the signature of the logging method (String pattern, Object.. parameters) (same as String.format())